Documentation restructure (#3050)
hyperspy/hyperspy: doc/intro.rst

described as multidimensional arrays of a given signal (e.g. a 2D array of
spectra a.k.a spectrum image).

HyperSpy aims at making it easy and natural to apply analytical procedures
that operate on an individual signal to multidimensional datasets of any
size, as well as providing easy access to analytical tools that exploit their
multidimensionality.

.. versionadded:: 1.5
  External packages can extend HyperSpy by registering signals,
  components and widgets.

.. _hyperspy_extensions-label:

External packages can extend HyperSpy to e.g. implement features to analyse a
particular sort of data. For details on how to register extensions see
:ref:`writing_extensions-label`. For a list of packages that extend HyperSpy
follow `this link <https://github.com/hyperspy/hyperspy-extensions-list>`_.

.. note::
    From version 2.0, HyperSpy will be split into a core package (HyperSpy)
    that will provide the common infrastructure and a number of HyperSpy
    extensions specialized in the analysis of different types of data.

HyperSpy's character
====================

HyperSpy has been written by a subset of the people who use it, a particularity
that sets its character:

> * To us this program is a research tool, much like a screwdriver or a Green's

Reviewer: ```suggestion
* To us, this program is a research tool, much like a screwdriver or a Green's
```

  packages for details. Not enough? If you
  need a full, standalone GUI, `HyperSpyUI <https://hyperspy.org/hyperspyUI/>`_
  is for you.

Learning resources
==================

.. grid:: 2 3 3 3
  :gutter: 2

  .. grid-item-card::

>     :link: user_guide/index.rst

Reviewer 2: ```suggestion
    :link: user_guide/index.html
```
At least on the `Read the Docs build`, linking to the `.rst` does not work. It gives a `404` error.

Reviewer 2: Currently the "Getting started" and the "User guide" card links to the same page: `user_guide/index.html`. Should:
- `Getting started` link to `user_guide/install.html`?
- `User guide` link to `user_guide/index.html`?

    :octicon:`rocket;2em;sd-text-info` Getting Started
    ^^^

    New to HyperSpy or Python? The getting started guide provides an

>     introduction on basic usage of HyperSpy and how to install it

Reviewer: ```suggestion
    introduction on basic usage of HyperSpy and how to install it.
```

  .. grid-item-card::
    :link: user_guide/index

    :octicon:`book;2em;sd-text-info` User Guide
    ^^^

    The user guide provides in-depth information on key concepts of HyperSpy

>     and how to use it along with background information and explanation

Reviewer: ```suggestion
    and how to use it along with background information and explanations.
```

  .. grid-item-card::

>     :link: reference/index.rst

Reviewer 2: ```suggestion
    :link: reference/index.html
```

    :octicon:`code-square;2em;sd-text-info` Reference
    ^^^

    Documentation of the metadata specification and of the Application Progamming Interface (API),
    which describe how HyperSpy functions work and which parameters can be used.

  .. grid-item-card::

>     :link: auto_examples/index.rst

Reviewer 2: ```suggestion
    :link: auto_examples/index.html
```

    :octicon:`zap;2em;sd-text-info` Example
    ^^^

>     Short examples illustrating simple tasks

Reviewer: ```suggestion
    Short examples illustrating simple tasks.
```

  .. grid-item-card::
    :link: https://github.com/hyperspy/hyperspy-demos

    :octicon:`workflow;2em;sd-text-info` Tutorials
    ^^^

>     Tutorials in form of Jupyter Notebook to learn how to

Reviewer: ```suggestion
    Tutorials in form of Jupyter Notebooks to learn how to
```

>     process multi-dimensional data using HyperSpy

Reviewer: ```suggestion
    process multi-dimensional data using HyperSpy.
```

  .. grid-item-card::

>     :link: dev_guide/index.rst

Reviewer 2: ```suggestion
    :link: dev_guide/index.html
```

    :octicon:`people;2em;sd-text-info` Contributing
    ^^^

    HyperSpy is a community project maintained for and by its users.
    There are many ways you can help!

Citing HyperSpy
================

If HyperSpy has been significant to a project that leads to an academic
publication, please acknowledge that fact by citing it. The DOI in the
badge below is the `Concept DOI <https://help.zenodo.org/#versioning>`_ of
HyperSpy. It can be used to cite the project without referring to a specific
version. If you are citing HyperSpy because you have used it to process data,
please use the DOI of the specific version that you have employed. You can
find iy by clicking on the DOI badge below.

.. image:: https://zenodo.org/badge/doi/10.5281/zenodo.592838.svg
   :target: https://doi.org/10.5281/zenodo.592838

HyperSpy's citation in the scientific literature
------------------------------------------------

> Given the increasing number of articles that cite HyperSpy we don't maintain a list of

Reviewer: ```suggestion
Given the increasing number of articles that cite HyperSpy, we do not maintain a list of
```

articles citing HyperSpy. For an up to date list search for
HyperSpy in a scientific database e.g. `Google Scholar
<https://scholar.google.co.uk/scholar?q=hyperspy>`_.

.. Warning::

>     The articles published before 2012 may mention the HyperSpy project under

Reviewer: ```suggestion
    Articles published before 2012 may mention the HyperSpy project under
```
